project/LibraryVersions.scala

```diff
@@ -5,4 +5,5 @@ object LibraryVersions {
   val catsVersion = "1.4.0"
   val jacksonVersion = "2.10.1"
   val okhttpVersion = "3.10.0"
+  val scalaUriVersion = "2.2.2"
```

Comment:
we were using different versions between workers and frontend, neither quite up to date (workers was pulling in an old domain of simulacrum which caused undetected binary incompatibilities - clashing class names)